src/randomNumbers.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = function generationRandomDigits() {
+  let result = '';
+
+  for (let i = 0; i < 4; i++) {
+    result += `${Math.floor(Math.floor(Math.random() * 10))}`;
```

Comment:
Nested Math.floor is unnecessary, use one Math.floor to round the random number.